data/containers/app.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+##
+## Simple Flask test program (see https://flask.palletsprojects.com/en/2.0.x/quickstart/)
+## Serve 'index.html' from the `templates` directory
+##
+## Usage: python3 app.py
+## Bind to port 80 or to the PORT environment variable, if set
+from flask import Flask, render_template
+import os, sys, ssl
+
+app = Flask(__name__)
+message = ""
+
+@app.route("/")
+def index():
+    return render_template("index.html")
+
+if __name__ == "__main__":
+    if len(sys.argv) > 1:
```

Comment:
if `message` is not needed in line 16, the lines 19-30 are useless, right? See my previous comment.

Sorry for the nitpicking :slightly_smiling_face:

Reply:
nope. this is irrelevant. 
`ENTRYPOINT ["python", "./app.py", "www.google.com"]` in dockerfile should explain it for you. which is used to `curl` later.

Reply:
Yes, and the outcome of this `curl` call is then stored in the `message` variable, which is never used.

You could replace `www.google.com` in the `ENTRYPOINT` with some bogus domain and the container application would still run successfully, e.g.

    ENTRYPOINT ["python", "./app.py", "http://www.thisdomaindoesnot.exist/"]

Here the container would behave just the same as with `www.google.com`. In the current state, this argument is irrelevant, and that makes me wonder why it is even there in the first place.

Reply:
We already have `containers/index.html` so please just read it and return it over HTTP. No need for any parameter here.

Reply:
i removed a lot of the confusion. mark as `this change is ready for review`

Reply:
https://openqa.opensuse.org/tests/1872711#